Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -695,16 +698,19 @@ def upload_file(entry_id: str, description: str) -> Tuple:
 
     file_ = demisto.getFilePath(entry_id)
     file_name = file_.get('name')  # pylint: disable=E1101
+    temp_file = open(file_.get('path'), 'rb')  # pylint: disable=E1101
```

Comment:
use `with` statement (better)
or add a try-finally scope so even in case of an error the file will be closed

same for both cases